### PR TITLE
remove ISRs for now

### PR DESCRIPTION
**Pull request by liamsi**
- leave placeholder in proto files
- comment out NID

## Description

_Please add a description of the changes that this PR introduces and the files that
are the most critical to review._ 

Closes: #649

### CODE REVIEW

**Comment by liamsi** [comment]
Proto-breakage action is expected to fail as we regenerated the proto files (and removed a field).

**Review by adlerjohn — APPROVED** [review]

File: proto/tendermint/types/types.proto [hunk 1]
```diff
@@ -88,7 +88,7 @@ message Data {
   // This means that block.AppHash does not include these txs.
   repeated bytes txs = 1;
 
-  IntermediateStateRoots intermediate_state_roots = 2 [(gogoproto.nullable) = false];
+  // field number 2 is reserved for intermediate state roots
```
Comment (evan-forbes) [inline proto/tendermint/types/types.proto:91]: :+1:

File: state/tx_filter_test.go [hunk 2]
```diff
@@ -26,7 +26,7 @@ func TestTxFilter(t *testing.T) {
 		isErr bool
 	}{
 		{types.Tx(tmrand.Bytes(2149)), false},
-		{types.Tx(tmrand.Bytes(2150)), true},
+		{types.Tx(tmrand.Bytes(2152)), true},
```
Comment (evan-forbes) [inline state/tx_filter_test.go:29]: I'm not 100% sure about this, but I think it makes sense after a bit of digging

**Review by evan-forbes — APPROVED** [review]
yeeting ISRs

**Pull request merged**